services/github/github-checks-status.service.js

```diff
@@ -0,0 +1,99 @@
+'use strict'
+
+const Joi = require('joi')
+const { NotFound, InvalidParameter } = require('..')
+const { GithubAuthV3Service } = require('./github-auth-service')
+const { documentation, errorMessagesFor } = require('./github-helpers')
+
+const schema = Joi.object({
+  state: Joi.equal('failure', 'pending', 'success'),
```

Comment:
Given this is essentially a canonical build-type badge, we should use the standard build badge helpers for the schema and rendering here. That will ensure that this badge is consistent with the others in the category, and also handle any additional states that may pop up if there's any changes upstream

these helpers can be imported from `build-status` located in the services directory (`const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')`

```suggestion
  state: isBuildStatus,
```